### PR TITLE
feat: add color fade function and `RBG`

### PR DESCRIPTION
**Pull request by floaust**
### Description
Added a color fade function and `RBG`.

- [ ] Bug Fix
- [x] New Feature
- [ ] Documentation
- [ ] Other

### Related Issue
Closes: #29

### CODE REVIEW

File: internal/map_range_to_range_test.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package internal
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestMapRangeToRange(t *testing.T) {
```
Comment (MarvinJWendt) [inline internal/map_range_to_range_test.go:8]: Please add some more tests with negative values, and float values (123.5, 123.324, etc.)

**Review by MarvinJWendt — CHANGES_REQUESTED** [review]

**Review by MarvinJWendt — APPROVED** [review]
LGTM! 😄
